CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ Use IntelliJ IDEA Community Edition/Ultimate.
    Open `Search Actions` dialog with Ctrl+Shift+A (⌘⇧A on Mac),
    type `plugins`, press Alt+Enter (⌥↩ on Mac), then press the chosen shortcut (suggested: Ctrl+Alt+Shift+P, or ⌘⌥⇧P on Mac).
 
-2. Make sure the following bundled plugins are enabled, in the `Installed` tab of `Preferences -> Plugins`:
+2. Make sure the following bundled plugins are enabled, in the `Installed` tab of `File > Settings > Plugins`:
```

Comment:
This change seems invalid to me. I'm using IntelliJ IDEA 2022.1 on macOs and I don't have anything like `File > Settings > Plugins`. Instead I have `Preferences -> Plugins` in my IDE. Maybe it is different for Mac vs Linux?
<img width="1176" alt="image" src="https://user-images.githubusercontent.com/49597713/193531786-2fda3b7d-9cfa-4be0-a0de-0417be8b47d2.png">

Reply:
yup, @PawelLipski if we are providing this path pls contain macOS steps too